modules/using-oci-containers-for-model-storage.adoc

```diff
@@ -0,0 +1,202 @@
+curl -L $DOWNLOAD_URL -O --output-dir models/1/
+----
++
+[NOTE]
+====
+The subdirectory `1` is used because OpenVINO requires numbered subdirectories for model versioning. If you are not using OpenVINO, you do not need to create the `1` subdirectory to use OCI container images.
+====
+. Create a Docker file named `Containerfile` with the following contents:
++
+[source]
+----
+FROM registry.access.redhat.com/ubi8/ubi-micro:latest
+COPY --chown=0:0 models /models
+RUN chmod -R a=rX /models
+
+# nobody user
+USER 65534 
+----
++
+[NOTE]
+====
+* In this example, `ubi8-micro` is used as a base container image. You cannot use empty images that do not provide a shell, such as `scratch`, because KServe uses the shell to ensure the model files are accessible to the model server. 
+* Ownership of the copied model files and read permissions are granted to the `root` group. OpenShift runs containers with a random user ID and the `root` group ID. Changing ownership of the group ensures that the model server can access them.
+====
++
+. Confirm that the models follow the directory structure shown using the `tree` command:
++
+[source]
+----
+tree
+
+.
+├── Containerfile
+└── models
+    └── 1
+        └── mobilenetv2-7.onnx
+----
++
+. Create the OCI container image with Podman, and upload it to a registry. For
+example, using Quay as the registry:
++
+[source]
+----
+podman build --format=oci -t quay.io/<user_name>/<repository_name>:<tag_name> .
+podman push quay.io/<user_name>/<repository_name>:<tag_name>
+----
++
+[NOTE]
+====
+If your repository is private, ensure you are authenticated to the registry before uploading your container image.
+====
+
+== Deploying a model stored in an OCI image in a public repository
+
+[NOTE]
+====
+By default in KServe, models are exposed outside the cluster and not protected with authorization. 
+====
+
+. Create a namespace to deploy the model:
++
+[source]
+----
+oc new-project oci-model-example
+----
++
+. Use the OpenData Hub project `kserve-ovms` template to create a `ServingRuntime` resource and configure the OpenVINO model server in the new namespace:
++
+[source]
+----
+oc process -n opendatahub -o yaml kserve-ovms | oc apply -f -
+----
++
+--
+.. Verify that the `ServingRuntime` has been created with the `kserve-ovms` name:
++
+[source]
+----
+oc get servingruntimes
+
+NAME          DISABLED   MODELTYPE     CONTAINERS         AGE
+kserve-ovms              openvino_ir   kserve-container   1m
+----
+--
++
+. Create an `InferenceService` YAML resource with the following values:
++
+[source]
+----
+apiVersion: serving.kserve.io/v1beta1
+kind: InferenceService
+metadata:
+  name: sample-isvc-using-oci
+spec:
+  predictor:
+    model:
+      runtime: kserve-ovms # Ensure this matches the name of the ServingRuntime resource
+      modelFormat:
+        name: onnx
+      storageUri: oci://quay.io/<user_name>/<repository_name>:<tag_name>
+----
++
+[NOTE]
```

Comment:
FWIW, this is more on the warning side, than a simple note.

Reply:
I bumped it up to an "Important" tag :)